tests/test_streaming_download_manager.py

```diff
@@ -29,36 +37,33 @@
     ],
```

Comment:
maybe we can add a test with local paths (both relative and absolute)
```suggestion
        (
            ".",
            ("file.txt",),
            "file.txt",
        ),
        (
            Path().resolve().as_posix(),
            ("file.txt",),
            (Path().resolve() / "file.txt").as_posix(),
        ),
    ],
```